WordPress/src/main/res/values/strings.xml

```diff
@@ -4697,4 +4697,26 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="partial_media_access_prompt_text">Only selected photos and videos you’ve given access to are available.</string>
     <string name="partial_media_access_prompt_select_more">Select More</string>
     <string name="partial_media_access_prompt_change_settings">Change Settings</string>
+
+    <!-- Personalisation Screen -->
+    <string name="personalisation_screen_title"> Personalise home tab</string>
```

Comment:
@AjeshRPai - Do we want to use us-en here or british? I know we have en-rGB, but is strings.xml the base. If so, do we want to use the american spelling "personalize" and not "personalise"?

Reply:
Yeah I agree, I will make it `personalise`. Also I think we should align on the package naming as well. 😄 . I went with `personalisation` on the packages, I can see that you went with `personalization`. 

I will rename packages and user facing strings to 'personalization' in a future PR.